src/test/java/bio/terra/common/BQTestUtils.java

```diff
@@ -0,0 +1,70 @@
+package bio.terra.common;
+
+import bio.terra.service.tabulardata.google.BigQueryProject;
+import com.google.cloud.PageImpl;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.FieldValue;
+import com.google.cloud.bigquery.FieldValueList;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.Schema;
+import com.google.cloud.bigquery.TableResult;
+import org.mockito.stubbing.Answer;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.mockito.Mockito.when;
+
+/**
+ * Helper methods for mocking out BigQuery responses
+ */
+public final class BQTestUtils {
+
+    private BQTestUtils() {
+    }
+
+    public static void mockBQQuery(BigQueryProject mockBQProject,
+                                   String sql,
+                                   Schema schema,
+                                   List<Map<String, String>> results
+    ) {
+        try {
+            when(mockBQProject.query(sql)).thenAnswer(mockAnswer(schema, results));
+        } catch (InterruptedException e) {
+            throw new RuntimeException("Error mocking query execution");
+        }
```

Comment:
In general, I don't try/catch in test code. In most cases (like this) exceptions are fatal and we want to report them at the line they occur on. So there's no benefit to using try/catch, in most cases.

 I also don't think it's worth trying to declare proper `throws` clauses. I think `throws Exception` is fine in test code, as we don't want to bother editing throws clauses because some API is now throwing a different exception.

Reply:
Yeah, in this case I just wanted to silence the exception to make it easier to pull in.  It shouldn't ever actually hit this branch

Reply:
But if it's only called from test code, all that code could declare `throws Exception`. If you don't take it out, at least include the caught exception in the throw so the underlying cause is in the stack crawl.